CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 *  Updated stencils and unit tests to pass [SwiftLint](https://github.com/realm/SwiftLint).  
   [Adam Gask](https://github.com/AJ9), [#79](https://github.com/AliSoftware/SwiftGen/pull/79)
+* Added support for `*.clr` files.
```

Comment:
Maybe explain a bit what those files are here. I wasn't aware of this files extension before, and just called them "system color palettes" so that "`.*.clr` files" didn't ring a bell to me first.
Just adding "(System Color Palettes)" should help — maybe a link to the the README too?